src/client/components/molecules/homeSections/second/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { useContext, useEffect, useState } from 'react';
+import ItemList from '../../../atoms/itemList';
+import './style.scss';
+import StateContext from '../../../../utils/stateContext';
+import { Portfolio } from '../../../../utils/Type';
+
+const SecondSection = () => {
+  const { handlers } = useContext(StateContext);
+  const [portfolioItems, setPortfolioItems] = useState<Portfolio[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      const response = await handlers.getAllPortfolios();
+      console.log('lola', response);
+
+      setPortfolioItems(response);
+    })();
+  }, []);
+
+  return (
+    <div className="secondSection">

```

Comment:
wrong use of BEM pattern - we don't use camelCase in classNames

the block should be something like `secont-section` (although is better if the block's className is just one word)